app/services/paypal_service/store/paypal_account.rb

```diff
@@ -103,16 +127,23 @@ def delete_billing_agreement_by_payer_and_agreement_id(payer_id:, billing_agreem
     maybe_billing_agreement.each { |billing_agreement| billing_agreement.destroy }
   end
 
-  def delete(person_id:nil, community_id:)
-    find_model(person_id: person_id, community_id: community_id).each { |account| account.destroy }
+  def delete(person_id:nil, community_id:, order_permission_request_token: :all)
+    find_model(person_id: person_id, community_id: community_id, order_permission_request_token: order_permission_request_token).each { |account| account.destroy }
   end
 
-  def get(person_id:nil, community_id:)
-    from_model(find_model(person_id: person_id, community_id: community_id))
+  def get(person_id:nil, community_id:, active: :all, payer_id: :all)
```

Comment:
This method signature is starting to feel like too many different use cases crammed into one shape. What if we'd have:

```ruby
def get(person_id:, community_id: nil, payer_id:)
def get_active(person_id:, community_id: nil)
```

So either you are looking for a specific account with all three parameters and don't care if it's active or not (inside transaction process) or then you just need the account to use for a person / community to start a transaction or to show in UI. Would that cover it?